Statistics/Distribution/NegativeBinomial.hs

```diff
@@ -151,7 +151,7 @@ complCumulative (NBD r p) x
   | k < 0        = 1
   | otherwise    = incompleteBeta (fromIntegral (k+1)) r (1 - p)
   where
-    k = (floor x)::Integer
+    k = floor x :: Integer
```

Comment:
Just an hlint warning fix here (note that `k = floor x :: Integer` is already used on line 145 above, so this makes the code a bit more consistent.